migrations/script.py.mako

```diff
@@ -0,0 +1,24 @@
+"""${message}
```

Comment:
Don't forget to leave a message in your `flask db migrate -m "A message for what changes you made to the model"`!